templates/mattermost-config.json.j2

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
why is this written like that? yaml is equivalent to json and they can be converted using e.g. the `to_json' filter.